### DISCUSSION TITLE
Increase the size of check in a circle and other instances

### DISCUSSION THREAD

**Discussion by zoharma** [lucide-icons/lucide#1864] [General]
Hi, wasn't sure where to place this, so thought this might be good as a thread.

Currently,  the `check-circle` icon looks smaller compared to other icons within a circle. 

Could we look at increasing it's size slightly, from `8px` wide to `10px` wide?
Although the `x-circle` is also 8px, it looks good in that size, but potentially can be looked at as well.

<img width="1456" alt="image" src="https://github.com/lucide-icons/lucide/assets/10050573/5d6c4324-8ec7-4ff9-a67c-c5fb28b3c417">

**Comment by heryckmp** [comment]
so fireeee

**Comment by karsa-mistmere** [comment]
I'm almost 100% sure we already have an issue or even PR for this. Yup, here it is: https://github.com/lucide-icons/lucide/pull/2402#issuecomment-2346152533
